client/src/components/widgets/ContextMenu/template.html

```diff
@@ -12,6 +12,11 @@
         <v-list-item dense @click="showRangeDialog=true">
           <v-list-item-title> Set global range </v-list-item-title>
         </v-list-item>
+        <v-list-item dense @click="toggleLogScale()">
```

Comment:
Either work but the following will prevent an extra wrap`@click="toggleLogScale"`

Reply:
unless you need to fully ignore method args

Reply:
No, no need to ignore args. Fixed.